src/content/advanced/labelling-workload-clusters/index.md

```diff
@@ -13,200 +13,84 @@ aliases:
 ```
 
-Additionally, [`kubectl label`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#label) can be used to modify workload cluster labels.
+## Modifying cluster labels
 
-```nohighlight
-kubectl label clusters.cluster.x-k8s.io/7g4di my-org/team=upstate
-```
+To modify labels on a cluster, several methods are supported:
 
-### Show workload cluster labels
+- Our [web UI]({{< relref "/ui-api/web/workload-cluster-labelling" >}}) allows adding, modifying, and deleting labels interactively.
+- The `kubectl label` command can be used with the cluster resource. See below for details. See below for details.
 
-Differing from `gsctl`, listing workload cluster labels with `kubectl` will show management labels required for operation. These usually contain `giantswarm.io` in its label keys and cannot be changed.
+### Modify cluster labels using `kubectl` {#modify-using-kubectl}
 
-Labels of all workload clusters:
+With access to the management cluster, you are able to use `kubectl` to manage workload cluster labels.
+The underlying resource to operate on is [`clusters.cluster.x-k8s.io`]({{< relref "/ui-api/management-api/crd/clusters.cluster.x-k8s.io.md" >}}) from the upstream [cluster-api](https://cluster-api.sigs.k8s.io/) project.
 
-```nohighlight
-$ kubectl labels --list --all clusters.cluster.x-k8s.io
-Listing labels for Cluster.cluster.x-k8s.io/7g4di:
- giantswarm.io/cluster=7g4di
- giantswarm.io/organization=my-org
- release.giantswarm.io/version=11.2.0
- cluster-operator.giantswarm.io/version=2.1.9
- my-org/team=upstate
- my-org/environment=testing
-Listing labels for Cluster.cluster.x-k8s.io/zv86a:
- cluster-operator.giantswarm.io/version=2.1.9
- giantswarm.io/cluster=q84ct
- giantswarm.io/organization=my-org
- release.giantswarm.io/version=11.3.0
- my-org/team=upstate
- my-org/environment=production
-```
+The [`kubectl label`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#label) command is the most convenient way to set, modify, and delete labels of a cluster resource.
 
-Labels of a single workload cluster:
+The following example shows how to set a label `my-org/team` with value `upstate` on a cluster named `7g4di`:
 
 ```nohighlight
-$ kubectl labels --list clusters.cluster.x-k8s.io/7g4di
-giantswarm.io/cluster=7g4di
-giantswarm.io/organization=my-org
-release.giantswarm.io/version=11.2.0
-cluster-operator.giantswarm.io/version=2.1.9
-my-org/team=upstate
-my-org/environment=testing
+kubectl label clusters.cluster.x-k8s.io/7g4di my-org/team=upstate
 ```
 
-### Select workload clusters by label selector
-
-Many `kubectl` commands support the `-l, --selector` flag, which allows to limit the selected resources based on given [Kubernetes Label selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors).
+Note that if the label already exists on the cluster, the `--overwrite` flag must be set. Otherwise the attempt to overwrite a label will result in n error.
```

Comment:
```suggestion
Note that if the label already exists on the cluster, the `--overwrite` flag must be set. Otherwise the attempt to overwrite a label will result in an error.
```